gpkit/tests/t_model.py

```diff
@@ -665,7 +686,12 @@ class TestModelNoSolve(unittest.TestCase):
     def test_modelname_added(self):
         t = Thing(2)
         for vk in t.varkeys:
-            self.assertEqual(vk.models, ["Thing"])
+            self.assertEqual(vk.lineage, (("Thing", 0),))
+
+    def test_modelcontainmentprinting(self):
+        t = Thing2()
+        self.assertIsInstance(t.str_without(), str)
+        self.assertIsInstance(t.latex(), str)
 
     def test_no_naming_on_var_access(self):
         # make sure that analysis models don't add their names to
```

Comment:
really happy that this big PR makes minimal, only necessary, changes to the tests and in fact adds tests. Great work